dockers/docker-dhcp-relay/cli-plugin-tests/mock_tables.py

```diff
@@ -146,6 +156,7 @@ def keys(self, pattern='*'):
 
 
 swsssdk.interface.DBInterface._subscribe_keyspace_notification = _subscribe_keyspace_notification
+swsssdk.interface.DBInterface.keys = keys
```

Comment:
are we still using swsssdk? @qiluo-msft , i thought swsssdk has been deprecated already.

Reply:
Seems like the `swsssdk` only be used in test. (macsec test use same approach as well: https://github.com/sonic-net/sonic-buildimage/blob/40834d51ee9b31ebf89b58b176b7fa7773d6d3f3/dockers/docker-macsec/cli-plugin-tests/mock_tables.py#L8)
We are using `swsscommon` in code implementation.

@yaqiangz for vis, can we use swsscommon in test?

Reply:
swsssdk is used in mock dbconnector in CLI ut (sonic-utilities / dhcp_relay / dhcp_server / macsec). From code, seems the purpose of them is to mock dbconnector in swsscommon with swsssdk
```
from swsssdk import SonicDBConfig, SonicV2Connector, ConfigDBConnector, ConfigDBPipeConnector
from swsscommon import swsscommon
...
swsscommon.SonicV2Connector = SonicV2Connector
swsscommon.ConfigDBConnector = ConfigDBConnector
swsscommon.ConfigDBPipeConnector = ConfigDBPipeConnector
```
And these changes were introduced when replacing swsssdk with swsscommon. swsssdk in running code has been removed, but in ut it is preserved for mocking. @qiluo-msft could you please help to check whether it's expected and whether we should remove swsssdk in CLI ut?
https://github.com/sonic-net/sonic-utilities/pull/1217/files#diff-cbaa80f38565f63f1f982f2220cc6e90d5a0b21028604886d87bd77f093e1ba5
https://github.com/sonic-net/sonic-utilities/pull/1392/files#diff-cbaa80f38565f63f1f982f2220cc6e90d5a0b21028604886d87bd77f093e1ba5